troubleshooting/first_aid_kit/index.rst

```diff
@@ -0,0 +1,142 @@
+First aid kit
+=============
+
+Here is our first aid kit if you encounter a bug on your PIM project.
+
+.. image:: /troubleshooting/unicorn-akeneo_128.png
+
+
+Step 1: are you sure that it's a bug?
+-------------------------------------
+
+Sometimes, it's quite clear that the behaviour you experience is a bug. But it is not always the case. Some behaviours can occur because you missed something (for instance a permissions misconfiguration), and the PIM just works as designed.
+
+If you have doubts, please have a look at the following resources. If not, go to the next step.
+
+* The PIM knowledge base: https://www.akeneo.com/knowledge-base/
+* The PIM user guides: https://www.akeneo.com/user-guide/
+
+.. tip::
+
+    User guides can be accessed at any moment from the **(?)** icon on the top right corner of the PIM.
+
+Step 2: is your computer ready to use the PIM?
+----------------------------------------------
+
+There are some client side requirements to use the PIM. Please check that everything is OK by looking at :doc:`/reference/technical_information/client_side_compatibilities`.
+
+Step 3: are the system requirements still OK?
+---------------------------------------------
+
+Maybe something has been changed in your server. You can check that by running the following command:
+
+.. code-block:: txt
+
+    cd /path/to/your/pim/
+    php app/console pim:installer:check-requirements
+
+.. note::
+
+    All our system requirements can also be found in this documentation:
+    :doc:`/developer_guide/installation/system_requirements/system_requirements`.
+
+In addition, check that:
+
+* The Xdebug PHP extension is well deactivated.
+* The PIM commands are well run in prod mode instead of dev mode (See: https://symfony.com/doc/2.7/configuration/environments.html#selecting-the-environment-for-console-commands)
+* Your crontab scheduled PIM commands are not launched too frequently.
```

Comment:
Maybe a link to https://docs.akeneo.com/latest/developer_guide/installation/installation_ce_archive.html#configuring-tasks-via-crontab would be nice